image-diff

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+
+from machine import testvm
+
+_, old_img, new_img = sys.argv
```

Comment:
Not the most friendly failure mode, but as that's a developer tool, it may be acceptable  :grin: Maybe add an `if len(sys.argv) != 2:` help message, or use `argparse` if you want to get fancy :)

Reply:
I wanna get fancy.  This is still a draft for a reason :)